fe/fe-core/src/main/java/com/starrocks/planner/IcebergTableSink.java

```diff
@@ -74,7 +76,16 @@ public IcebergTableSink(IcebergTable icebergTable, TupleDescriptor desc, boolean
         Connector connector = GlobalStateMgr.getCurrentState().getConnectorMgr().getConnector(catalogName);
         Preconditions.checkState(connector != null,
                 String.format("connector of catalog %s should not be null", catalogName));
-        this.cloudConfiguration = connector.getCloudConfiguration();
+
+        // Try to set for tabular
+        CloudConfiguration tabularTempCloudConfiguration = CloudConfigurationFactory.
```

Comment:
You can consider checking if the 'tabular' option is available first and then build the cloud configuration accordingly.

Reply:
fixed